documentation/Doxyfile.cfg

```diff
@@ -2021,7 +2035,7 @@ RTF_EXTENSIONS_FILE    =
 
 GENERATE_MAN           = NO
 
-# The MAN_OUTPUT tag is used to specify where the man pages will be put. If a
+# The MAN_OUTPUT tag is used to specify where the man И will be put. If a
```

Comment:
Out of curiosity, why this change?

Reply:
looks like some replace artifact

Reply:
Oops, I think I just accidentally typed that, fixed now